### ISSUE TITLE
[css-forms-1] Use empty alt text for ::picker-icon and ::checkmark

### ISSUE THREAD

**Issue by fantasai** [w3c/csswg-drafts#14316]
The icons we use for ::checkmark and ::picker-icon are not particularly useful to read aloud. The status of the control is already exposed to AT independently, so we propose using empty alt text for these glyphs:

```
::checkmark {
  content: "\2713" / "";
}
::picker-icon {
  content: ... / "";
}
```

**Comment by josepharhar** [comment]
In chromium, we manually remove these pseudo elements from the accessibility tree so that its functionally the same as having empty alt text. If an author overrides this value, I don't think that they should have to remember to make the alt text empty like we could do in the UA stylesheet.

**Comment by josepharhar** [comment]
I'm not sure that this was expicitly said in the resolution, but we are not going to set the alt text in the content property in the UA stylesheet, right?

**Comment by matatk** [comment]
We looked at this in a recent Accessible Platform Architectures WG call - thanks for flagging it @fantasai.

We appreciate the intent behind this change, however, after discussing it, we think it may be better to take a different approach.

Right now, this change is effectively a no-op, because any alt text (accessible name) being provided to the glyph will be overridden by the accessible name provided to the button that contains it.

However, in future, it is possible that users and ATs may wish to delve inside the button, to inspect it in more detail. In this case, overriding the glyph to have no name actually _removes_ information.

Whilst no AT does this yet, it would be nice to not preclude it as an option in future, and it's possible that the richer info gleaned from the choice of glyph could be of interest or assistance.

As CSSWG resolved, we too are keen to hear ARIA WG's view on this.
